rpc/rpcl/block.x

```diff
@@ -36,6 +36,7 @@ struct blockCreate2 {
   string    block_hosts<>;               /* for multiple tpg's creation */
   bool      auth_mode;
   u_int     rb_size;                     /* TCMU Ring Buffer size in kernel */
+  u_int     blk_size;                    /* TCMU hw block size in kernel */
```

Comment:
Since this is on the server side we need a rpc structure blockCreate3 else the remote procedure which is at lower version won't be able to decode.

Reply:
Okay, make sense.